app/_hub/kong-inc/session/2.2-x.md

```diff
@@ -0,0 +1,403 @@
+  <a href="https://github.com/bungle/lua-resty-session">lua-resty-session</a>.
+
+  For information about configuring and using the Sessions plugin with the Dev
+  Portal, see [Sessions in the Dev Portal](/enterprise/latest/developer-portal/configuration/authentication/sessions/#configuration-to-use-the-sessions-plugin-with-the-dev-portal).
+
+type: plugin
+categories:
+  - authentication
+
+source_url: https://github.com/Kong/kong-plugin-session
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.0.x
+      - 1.5.x
+      - 1.4.x
+      - 1.3.x
+      - 1.2.x
+  enterprise_edition:
+    compatible:
+      - 1.5.x
+      - 1.3-x
+      - 0.36-x
+      - 0.35-x
+
+params:
+  name: session
+  service_id: true
+  route_id: true
+  consumer_id: false
+  config:
+    - name: secret
+      required: false
+      default: random number generated from `kong.utils.random_string`
+      value_in_examples: opensesame
+      description: The secret that is used in keyed HMAC generation.​
+    - name: cookie_name
+      required: false
+      default: '`session`'
+      description: The name of the cookie
+    - name: cookie_lifetime
+      required: false
+      default: 3600
+      description: The duration (in seconds) that the session will remain open
+    - name: cookie_renew
+      required: false
+      default: 600
+      description: The duration (in seconds) of a session remaining at which point the Plugin renews the session
+    - name: cookie_path
+      required: false
+      default: '/'
+      description: The resource in the host where the cookie is available
+    - name: cookie_domain
+      required: false
+      default: Set using Nginx variable host, but may be overridden
+      description: The domain with which the cookie is intended to be exchanged
+    - name: cookie_samesite
+      required: false
+      default: 'Strict'
+      description: 'Determines whether and how a cookie may be sent with cross-site requests. "Strict": the browser will send cookies only if the request originated from the website that set the cookie. "Lax": same-site cookies are withheld on cross-domain subrequests, but will be sent when a user navigates to the URL from an external site, for example, by following a link. "off": disables the same-site attribute so that a cookie may be sent with cross-site requests. https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#SameSite_cookies'
+    - name: cookie_httponly
+      required: false
+      default: true
+      description: Applies the `HttpOnly` tag so that the cookie is sent only to a server https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies
+    - name: cookie_secure
+      required: false
+      default: true
+      description: Applies the Secure directive so that the cookie may be sent to the server only with an encrypted request over the HTTPS protocol https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies
```

Comment:
```suggestion
      description: Applies the Secure directive so that the cookie may be sent to the server only with an encrypted request over the HTTPS protocol https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies
```

Reply:
Same as above, the anchor needs updating.